README.md

```diff
@@ -7,46 +7,45 @@
 
 Interpret Community SDK
 =============================================================
+[InterpretML](https://github.com/interpretml/) is a machine learning interpretability package for training interpretable machine learning models and/or explaining blackbox AI systems. InterpretML that consists of the following repositories:
 
+| Component | Description |
+| ---- | --- |
+| [**Interpret**](https://github.com/interpretml/interpret) | Supporting a collection of glassbox models and blackbox/graybox explainers for **tabular data** scenarios|
```

Comment:
It sounded like we need to start getting rid of "graybox" terminology

Reply:
Removed and changed to model-specific and model-agnostic.